src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -804,9 +806,16 @@
                   :in csr-stream
                   :env (merge env {"RUBYLIB" rubylib})})]
     (log/debugf "Autosign command '%s %s' exit status: %d"
-                executable subject (:exit results))
+                executable subject (:exit-code results))
     (log/debugf "Autosign command '%s %s' output: %s"
-                executable subject (str (:err results) (:out results)))
+                executable subject (str (:stderr results) (:stdout results)))
```

Comment:
Yeah, I think logging at :warn is a great idea for stderr. But if folks have to turn the logging up to debug this issue then they'll be getting stdout on debug and stderr on both debug and warn. Which I don't know if that's a good thing (really highlights stderr) or a bad thing (duplicated logs).